packages/wonder-blocks-banner/src/components/banner.tsx

```diff
@@ -302,40 +323,102 @@ const Banner = (props: Props): React.ReactElement => {
+                inlineEnd: border.width.none,
+                blockStart: border.width.none,
+                blockEnd: border.width.none,
+            },
+        },
+        layout: {
+            padding: sizing.size_080,
+        },
+        color: {
+            border: {
+                info: semanticColor.core.border.instructive.default,
+                success: semanticColor.core.border.success.default,
+                warning: semanticColor.core.border.warning.default,
+                critical: semanticColor.core.border.critical.default,
+            },
+            background: {
+                info: semanticColor.feedback.info.subtle.background,
+                success: semanticColor.feedback.success.subtle.background,
+                warning: semanticColor.feedback.warning.subtle.background,
+                critical: semanticColor.feedback.critical.subtle.background,
+            },
+        },
+    },
+    icon: {
+        color: {
+            info: semanticColor.icon.primary,
+            success: semanticColor.icon.primary,
+            warning: semanticColor.icon.primary,
+            critical: semanticColor.icon.primary,
+        },
+        layout: {
+            marginBlockStart: sizing.size_080,
+            marginBlockEnd: sizing.size_080,
+            // The total distance from the icon to the edge is 16px. The
+            // vertical identifier is already 6px, and the padding on inner
+            // conatiner is 8px. So that leaves 2px.
+            marginInlineStart: sizing.size_020,
+            marginInlineEnd: sizing.size_080,
+        },
     },
+    label: {
+        layout: {
+            margin: sizing.size_080,
+        },
+    },
+    actions: {
+        layout: {
+            marginBlock: sizing.size_080,
+        },
+        sizing: {
+            height: sizing.size_180,
+        },
+    },
+    action: {
+        layout: {
+            marginInline: sizing.size_080,
+        },
+    },
+    dismiss: {
+        sizing: {
+            height: sizing.size_400,
+            width: sizing.size_400,
+        },
+        layout: {
+            marginInline: sizing.size_080,
+        },
+    },
+};
+
+const styles = StyleSheet.create({
     containerOuter: {
-        borderInlineStartWidth: spacing.xxSmall_6,
+        borderInlineStartWidth: bannerTokens.root.border.width.inlineStart,
+        borderInlineEndWidth: bannerTokens.root.border.width.inlineEnd,
+        borderBlockStartWidth: bannerTokens.root.border.width.blockStart,
+        borderBlockEndWidth: bannerTokens.root.border.width.blockEnd,
         width: "100%",
-        // Because of the background color's opacity value,
-        // the base color needs to be hard-coded as white for the
-        // intended pastel background color to show up correctly
-        // on dark backgrounds.
-        // TODO(WB-1865): Verify if we can change this to use semanticColor
```

Comment:
We are able to use semantic colors for the background, without having to set the opacity!